.github/workflows/ci-sharktank.yml

```diff
@@ -22,6 +22,31 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  test_punet:
```

Comment:
We should also change model version to this (https://github.com/nod-ai/shark-ai/pull/569/files) to reflect the production model

Reply:
ah my bad good catch